docs/articles/apex/refactoring.md

```diff
@@ -15,3 +15,25 @@ Validation of the new symbol name is performed before applying the rename. If th
 If the new name conflicts with an existing identifier name, we try to fully qualify the references to the existing name in contexts where the conflicts exist.
 
 ![GIF showing a renaming conflict](../../images/apex-rename-conflict.gif)
+
+## Refactor: Extract Constant
+
+You can extract valid literal expressions into constants. Literal expressions include: String, Integer, Long, Double, Decimal, and Boolean.
```

Comment:
It is not clear to me what "valid literal expressions" mean. We can remove "valid" i think. Also, no need to specify "literal expressions", just "literals" should be fine. The wording her could be "Extract constant refactoring is allowed only on literals". If you do that, then there is no need to specify the types of the literal expressions.